test/functional/url-expander/test-expander.js

```diff
@@ -151,6 +151,7 @@ describes.realWin('Expander', {
       ASYNCFN: arg => Promise.resolve(arg),
       BROKEN: () => undefined,
       ANCESTOR_ORIGIN: () => Promise.resolve('https://www.google.com@foo'),
+      TITLE: 'hello world',
```

Comment:
add a trailing space. so we can test that only the out most macro does the encoding.